### ISSUE TITLE
Screenshot Mode

### ISSUE THREAD

**Issue by tylerjroach** [JakeWharton/Telecine#76]
What are your thoughts about a screenshot feature? This could exist as a toggle in the app between video recording vs. screenshot. When the overlay is launched in screenshot mode, a similar overlay could be launched. I'd imagine the overlay is immediately hidden when the capture button is pressed and reappears once the screenshot has been taken.

I have a friend who is partially paralyzed and has trouble using the hardware buttons to take screenshots. I planned on using this app as a base for a screenshot app (all the current ones on the market seem to be terrible). If it is something you may consider allowing into the app, I may just submit it as a pull request, rather than a separate app.

Thanks!

**Comment by tylerjroach** [comment]
I've gotten this working in my own repo. In trying to change the current code as little as possible, I created a separate ScreenshotOverlayView and ScreenshotSession. If you did want the screenshot feature to exist in Telecine, I guess the question would be how to best handle the recording vs. screenshot mode. I planned on having a drop down spinner with the selected mode, then show or hide the preferences depending on which mode was selected.

**Comment by JakeWharton** [comment]
Yep, that seems reasonable. This would be great to get in, and you're doing
it for an awesome reason!

**Comment by tylerjroach** [comment]
Awesome, I'll try to complete it tonight or tomorrow and send a pull request for review.

**Comment by marcardar** [comment]
On Android M, it might make more sense to use the Assistant for taking a screenshot: http://developer.android.com/training/articles/assistant.html

**Comment by tylerjroach** [comment]
I still haven't gotten the chance to take a look at Assistant, although I don't see many positives to implementing this in a way that is 23+ only. I'm still not 100% sure this pull request will be accepted as it may be a somewhat limited use case.

The request is here if you want to check it out: https://github.com/JakeWharton/Telecine/pull/77

**Comment by marcardar** [comment]
The huge advantage is that the user can take a screenshot without having any app running. It would make a screenshot mode/button redundant for 23+. The question then is whether a screenshot mode/button is worth it when it is only relevant for 21,22.

**Comment by M2ck** [comment]
This feature would be really usefull, looking for a such thing :+1:

**Comment by tylerjroach** [comment]
@M2ck If it is something you really want to see, you can git clone https://github.com/tylerjroach/Telecine.git. Although this fork may be behind on other features.

**Comment by M2ck** [comment]
@tylerjroach thanks for the tip but I'm not at all a developper, just an advanced user sorry.

**Comment by AndroidDeveloperLB** [comment]
@JakeWharton Any news about the screenshot feature?
Also, can you please check out this post:
http://stackoverflow.com/q/43705756/878126
